nailgun/entities.py

```diff
@@ -1105,6 +1105,10 @@ def path(self, which=None):
             return '{0}/promote'.format(
                 super(ContentViewVersion, self).path(which='self')
             )
+        if which == 'incremental_update':
+            return '{0}/incremental_update'.format(
+                super(ContentViewVersion, self).path(which=None)
+            )
```

Comment:
That won't work correctly. It'll produce a path like so:

    /katello/api/v2/content_view_versions/<id>/incremental_update

Which is incorrect. The current implementation will sometimes produce that path, too. `which` should be "base":

```python
super(ContentViewVersion, self).path(which='self'),
```